Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBiasHI_cff.py

```diff
@@ -26,7 +26,8 @@
 
 import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_ho_cfi
 
-seqALCARECOHcalCalMinBias = cms.Sequence(hcalminbiasHLT*hcalDigiAlCaMB*gtDigisAlCaMB*hbherecoNoise*hfrecoNoise*hfrecoMBNZS*horecoNoise)
+seqALCARECOHcalCalMinBiasDigi = cms.Sequence(hcalminbiasHLT*hcalDigiAlCaMB*gtDigisAlCaMB)
```

Comment:
It is needed because later on one updates one of the sequences. With a combined one it was doing the changes wrongly which was noticed by Olga Kodolova